### DISCUSSION TITLE
zone.php v23.03 no selection in picture

### DISCUSSION THREAD

**Discussion by alex001x** [opendcim/openDCIM#1460] [Q&A]
Hi,
impossible to create a selection zone on the picture ! Bug?

**Comment by wilpig** [comment]
Browser/os ? Does it work on dev.opendcim.org ?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Alexandre ***@***.***>
Sent: Wednesday, October 11, 2023 2:59:56 PM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] zone.php v23.03 no selection in picture (Discussion #1460)


Hi,
impossible to create a selection zone on the picture ! Bug?

—
Reply to this email directly, view it on GitHub<https://github.com/opendcim/openDCIM/discussions/1460>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAMTY6XHTVLO4S3AIWCA5CDX633DZANCNFSM6AAAAAA54Q36OE>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/opendcim/openDCIM/discussions/1460", "url": "https://github.com/opendcim/openDCIM/discussions/1460", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by alex001x** [reply]
OS : Ubuntu
Browser : Firefox
does not work on "https://dev.opendcim.org/zone.php?zoneid=5"

**Comment by wilpig** [comment]
I'll look into it tomorrow, I have plans tonight that won't permit time to dig. It's likely something related to the jQuery update, you can hit f12 for the debug console and see if anything super obvious pops out

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wilpig** [comment]
I've now tried both Firefox and Chrome on MacOS14 and concluded, this is not a bug and is some form of pebkac.
<img width="1057" alt="Screenshot 2023-10-11 at 6 03 09 PM" src="https://github.com/opendcim/openDCIM/assets/1653882/0265f5f6-3411-425d-b2b3-17a1d1bbc625">
https://demo.opendcim.org/dc_stats.php?dc=1
https://demo.opendcim.org/zone.php?zoneid=1

I was able to reshape a zone on the demo site. and I was able to create a new layout on the dev site.
<img width="1601" alt="Screenshot 2023-10-11 at 6 06 31 PM" src="https://github.com/opendcim/openDCIM/assets/1653882/54c6c3c7-24cd-40f1-ba4a-dcfa869de085">
https://dev.opendcim.org/zone.php?zoneid=5
